src/app/helptext/task-calendar/replication/replication-wizard.ts

```diff
@@ -54,25 +54,25 @@ export default {
  the selected dataset snapshots.'),
 
     encryption_placeholder: T('Encryption'),
-    encryption_tooltip: T(''),
+    encryption_tooltip: T('Checking this box, enables additional options for encryption replication'),
 
     encryption_key_generate_placeholder: T('Generate Encryption Key'),
-    encryption_key_generate_tooltip: T(''),
+    encryption_key_generate_tooltip: T('If Hex key type is selected, this will autogenerate an encryption key'),
```

Comment:
Cleanup suggestion:
"If the <i>Hex key</i> type is chosen, an encryption key will be autogenerated."